projects/plugins/boost/rollup.config.js

```diff
@@ -130,6 +131,8 @@ export default {
 		typescript( {
 			sourceMap: ! production,
 			inlineSources: ! production,
+			include: tsconfig.include,
```

Comment:
This is for the same reason as mentioned here https://github.com/Automattic/jetpack/pull/23889#pullrequestreview-941803395